tests/core/tools/manual/keystroke.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: feature, 4.9.0, 1338
```

Comment:
There are still stray bold, italics, underline buttons displayed. Instead you could simplify it by leaving just buttons that are really needed for the test.

You could use `config.removeButtons` for that.

Reply:
Also it would be nice to give this test more precise name. In the end it just checkes function keys, it's not a test for keystrokes in general.